### ISSUE TITLE
CMD CHECK --as-cran

### ISSUE THREAD

**Issue by kaizadp** [kaizadp/nmrrr#53]
@bpbond 

Got some errors and notes when running CMD CHECK --as-cran

```
* checking CRAN incoming feasibility ... NOTE
Maintainer: ‘Kaizad Patel <kaizad.patel@pnnl.gov>’
```
^ this is OK

```
* checking top-level files ... NOTE
Files ‘README.md’ or ‘NEWS.md’ cannot be checked without ‘pandoc’ being installed.
```
^ do we need to include `pandoc` in the dependencies?

```
* checking PDF version of manual ... WARNING
LaTeX errors when creating PDF version.
This typically indicates Rd problems.
* checking PDF version of manual without index ... ERROR
Re-running with no redirection of stdout/stderr.
* checking for non-standard things in the check directory ... NOTE
Found the following files/directories:
  ‘nmrrr-manual.tex’
```
^ I tried searching online, but can't find a solution!

```
Status: 1 ERROR, 1 WARNING, 3 NOTEs
```

This is the log:
[00check.log](https://github.com/kaizadp/nmrrr/files/11291914/00check.log)

**Comment by kaizadp** [comment]
I was also getting `Package inputenc Error: Unicode character α (U+3B1) not set up for use with LaTeX`, but it's gone now. Maybe because I installed `tinytex` package.

**Comment by kaizadp** [comment]
In my terminal, I get these additional messages:

```
* checking PDF version of manual ... WARNING
LaTeX errors when creating PDF version.
This typically indicates Rd problems.
* checking PDF version of manual without index ... ERROR
Re-running with no redirection of stdout/stderr.
Hmm ... looks like a package
Converting parsed Rd's to LaTeX .
Creating pdf output from LaTeX ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  pdflatex is not available
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  pdflatex is not available
Error in running tools::texi2pdf()
You may want to clean up by 'rm -Rf /var/folders/jf/ztxc08pd6p35l0_pwdssdjw00000gn/T//RtmpKoyWIs/Rd2pdf14b706baf64b9'
* checking for non-standard things in the check directory ... NOTE
Found the following files/directories:
  ‘nmrrr-manual.tex’
```

**Comment by bpbond** [comment]
I wouldn't worry about any of this. Your system isn't set up like CRAN, and obviously doesn't have some of the required tools; but if the package is fully passing the normal check process, it should be fine.

**Comment by kaizadp** [comment]
reinstalled `tinytex`. Gave a new error: `Package inputenc Error: Unicode character α (U+3B1) not set up for use with LaTeX`.

Used [this issue](https://github.com/r-lib/roxygen2/issues/750) to locate the problem - it was the α in some of the bin descriptions. fixed in 5322c24.
